infrastructure/modules/scaling_service/queue.tf

```diff
@@ -1,5 +1,5 @@
 module "input_queue" {
-  source = "github.com/wellcomecollection/terraform-aws-sqs//queue?ref=v1.4.0"
+  source = "github.com/wellcomecollection/terraform-aws-sqs//queue?ref=max-age-alarm"
```

Comment:
I will update this when https://github.com/wellcomecollection/terraform-aws-sqs/pull/14 is merged.